modules/dna_rna_tools.py

```diff
@@ -0,0 +1,68 @@
+complement_dict_dna = {
+    "A": "T",
+    "T": "A",
+    "G": "C",
+    "C": "G",
+    "a": "t",
+    "t": "a",
+    "g": "c",
+    "c": "g",
+}
+
+complement_dict_rna = {
+    "A": "U",
+    "U": "A",
+    "G": "C",
+    "C": "G",
+    "a": "u",
+    "u": "a",
+    "g": "c",
+    "c": "g",
+}
+
+
+def transcribe(lst_nucl_seq: str|list) -> str|list:
+    lst_trscr_out = []
+    for seq in lst_nucl_seq:
+        if "U" in seq or "u" in seq:
+            return "It is RNA"
+        else:
+            seq = seq.replace("T", "U").replace("t", "u")
+            lst_trscr_out.append(seq)
+    return lst_trscr_out if len(lst_trscr_out) > 1 else lst_trscr_out[0]
+
+
+def reverse(lst_nucl_seq: str|list) -> str|list:
+    lst_rev_out = []
+    for seq in range(len(lst_nucl_seq)):
+        lst_rev_out.append(lst_nucl_seq[seq][::-1])
+    return lst_rev_out if len(lst_rev_out) > 1 else lst_rev_out[0]
+
+
+def complement(lst_nucl_seq: str|list) -> str|list:
+    lst_compl_out = []
+    for seq in lst_nucl_seq:
+        complement_output = ""
+        if set(seq) <= set("ATGCatgc"):
+            for bp in seq:
+                complement_output += complement_dict_dna[bp]
+            lst_compl_out.append(complement_output)
+        elif set(seq) <= set("AUGCaugc"):
+            for bp in seq:
+                complement_output += complement_dict_rna[bp]
+            lst_compl_out.append(complement_output)
```

Comment:
здесь лучше вынести из цикла выбор словаря, тогда будет сильно проще
```suggestion
        if set(seq) <= set("ATGCatgc"):
            compl_dict = complement_dict_dna
        else:
            compl_dict = complement_dict_rna
        lst_compl_out.append("".join(compl_dict[bp] for bp in seq))
```